server/src/index.js

```diff
@@ -11,52 +11,83 @@ const url = process.env.URL
 app.listen(3000, () => console.log('Listening to port 3000'))
 app.get('/user/:id', (req, res) => res.send(`Welcome to the homepage of user ${req.params.id}.`))
 
+function createPost(req, type) {
+  const post = {}
+  switch (type) {
+    case 0:
+      post.title = req.body.title
```

Comment:
You need to check if the title is valid, otherwise people can submit empty title. You can use `if (req.body.title)`. If it's invalid, you should return 400.